### DISCUSSION TITLE
Enable code syntax highlighting when exporting to PDF

### DISCUSSION THREAD

**Discussion by ma-sadeghi** [jupyter-book/mystmd#1029] [Q&A]
When I export a myst-md file to pdf using `myst build`, the code syntax highlighting is gone in the PDF. Is there a way to enable it?

**Comment by 72757373656c6c** [comment]
If I use `make clean; make latexpdf`, the code blocks in the resulting pdf are highlighted.

If I use the theme's "Print to pdf" in the browser, when the browser's print dialog is displayed, I have to make sure to select Color Mode.

Versions: myst-parser 2.0.0, sphinx-book-theme 1.1.2, and Pygments 2.17.2

When I ran `sphinx-quickstart`, a `make` file was created that executes `sphinx-build`. I noticed you used `myst build`. I'm a little fuzzy :smile: on the relationships here, but I'm wonder if the answer might be in the doc, issues or discussion at mystmd.

- https://mystmd.org/guide/creating-pdf-documents 
- https://github.com/executablebooks/mystmd.
